README.md

```diff
@@ -204,8 +231,66 @@ other log management system if you wish to persist logs.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
+#### Windows Standalone .zip
+If you don't want to use the installer script, we offer a
+.zip that can be deployed to the target host.  This bundle is available for
+download on the [Github Releases
+Page](https://github.com/signalfx/signalfx-agent/releases) for each new
+release.
+
+Before proceeding please satisfy the following three requirements are installed
```

Comment:
satisfy the following three requirements are installed --> make sure the following three requirements are met